### PR TITLE
add cronjob for metrics at ansible

### PR DESCRIPTION
**Pull request by lionick**

### CODE REVIEW

File: roles/rciam-metrics/templates/fastapi/fastapi.service.j2 [hunk 1]
```diff
@@ -5,11 +5,11 @@ After=network.target
 [Service]
 User={{ metrics_user.name }}
 Group=www-data
-
+Restart=always
 WorkingDirectory={{ metrics_path }}
 Environment="PATH={{ metrics_path }}/.venv/bin"
-ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix {{ metrics_api.log_identifier }}-service --bind {{ lookup('dig', inventory_hostname) }} --log-level {{ metrics_api.log_level }}
-
+ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 8 -k uvicorn.workers.UvicornWorker app.main:app --bind 0.0.0.0:8000 --log-level {{ metrics_api.log_level }} --timeout {{ metrics_api.timeout }}
```
Comment (NicolasLiampotis) [inline roles/rciam-metrics/templates/fastapi/fastapi.service.j2:11]: Check if we need to bind to all interfaces or we can just bind to localhost. This depends on the vhost configuration in nginx

File: roles/rciam-metrics/tasks/configure.yml [hunk 2]
```diff
@@ -66,3 +66,17 @@
...
+    user: "{{ metrics_user.name }}"
+    cron_file: "{{ metrics_user.name}}"
+    job: "{{ metrics_user.cron.job }}"
+    minute: "{{ metrics_user.cron.minute | default(omit) }}"
+    hour: "{{ metrics_user.cron.hour | default(omit) }}"
+    day: "{{ metrics_user.cron.day | default(omit) }}"
+    month: "{{ metrics_user.cron.month | default(omit) }}"
+  when: metrics_user.cron is defined
+  become: yes
+  run_once: true
```
Comment (NicolasLiampotis) [inline roles/rciam-metrics/tasks/configure.yml:81]: use enabled flag from default role/hosts.ini config

**Comment by lionick** [comment]
@NicolasLiampotis I have done these changes from yesterday, please check.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**